molo/forms/migrations/0009_add_article_form_field.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.12 on 2020-05-05 08:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('forms', '0008_articlepageforms'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='moloformpage',
+            name='article_form_only',
+            field=models.BooleanField(default=False, help_text='This will display only in article pages', verbose_name='Is Article Form Only'),
+        ),
+        migrations.AlterField(
+            model_name='personalisableformfield',
+            name='field_type',
+            field=models.CharField(choices=[('singleline', 'Single line text'), ('multiline', 'Multi-line text'), ('email', 'Email'), ('number', 'Number'), ('url', 'URL'), ('checkbox', 'Checkbox'), ('checkboxes', 'Checkboxes'), ('dropdown', 'Drop down'), ('radio', 'Radio buttons'), ('date', 'Date'), ('datetime', 'Date/time'), ('hidden', 'Hidden field')], max_length=16, verbose_name='field type'),
```

Comment:
I don't see this change in the models.py. Is it supposed to be here?

Reply:
Thanks for this... will remove